rts/at/tx/src/main/java/org/jboss/jbossts/star/resource/Transaction.java

```diff
@@ -74,6 +74,16 @@ public Transaction(Uid uid) {
         volatileParticipants = new ArrayList<String>();
     }
 
+    @Override
+    public boolean equals(Object o) {
+        return super.equals(o);
```

Comment:
Yes it was. It is expecting Java equality (ie field by field) but equality of transactions is by uid. Maybe my adding it is overkill - I will delete it but remember to ignore the findbugs report.